Formula/rxcpp.rb

```diff
@@ -0,0 +1,62 @@
+class Rxcpp < Formula
+  desc "Reactive Extensions for C++"
+  homepage "https://rxcpp.codeplex.com/"
+  url "https://github.com/Reactive-Extensions/RxCpp.git",
+      :tag => "v3.0.0",
+      :revision => "8290f92f744f807e83b1bfe9e8c0ffd162140ec8"
+  head "https://github.com/Reactive-Extensions/RxCpp.git"
+
+  bottle :unneeded
+
+  option "with-test", "Build and run the test suite"
```

Comment:
to compile it it's rather long... on my mbp pro 2015 took about 20 mins to get the test suit compiled